Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -334,6 +334,12 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
             )
         )
 
+#if DD_SDK_ENABLE_INTERNAL_MONITORING
+        if #available(iOS 15, *) {
+            MetricMonitor.shared.monitorMetricKit(launchTime: dependencies.launchTimeProvider.launchTime)
+        }
+#endif
+
```

Comment:
The idea is to compare metrics with the SDK measure of the ALT, so I'me making sure sure that we have reported the ALT before starting the monitor. `RUMApplicationScope.init()` is probably too early in the process.

> That makes me think that this implementation does not log the delay between the registration and the reception of the payload!

Reply:
> The idea is to compare metrics with the SDK measure of the ALT, so I'm making sure sure that we have reported the ALT before starting the monitor. RUMApplicationScope.init() is probably too early in the process

I see 👍, this makes sense then, but I think it would be handy to add a little comment on this:
```swift
// Starting MetricKit from here, to ensure that our launch time was already reported in `.applicationStart` action and we could compare both measurements
MetricMonitor.shared.monitorMetricKit(launchTime: dependencies.launchTimeProvider.launchTime)
```